<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabGalpao</class>
 <widget class="QWidget" name="TabGalpao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGalpao">
      <attribute name="title">
       <string>Galp√£o</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="WidgetGalpao" name="widgetGalpao" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPeso">
      <attribute name="title">
       <string>Peso</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="WidgetGalpaoPeso" name="widgetGalpaoPeso" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetGalpaoPeso</class>
   <extends>QWidget</extends>
   <header>src/widgetgalpaopeso.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetGalpao</class>
   <extends>QWidget</extends>
   <header>src/widgetgalpao.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
